Doc/library/__future__.rst

```diff
@@ -34,36 +40,35 @@
    where, normally, *OptionalRelease* is less than *MandatoryRelease*, and both are
    5-tuples of the same form as :data:`sys.version_info`::
 
-      (PY_MAJOR_VERSION, # the 2 in 2.1.0a3; an int
-       PY_MINOR_VERSION, # the 1; an int
-       PY_MICRO_VERSION, # the 0; an int
-       PY_RELEASE_LEVEL, # "alpha", "beta", "candidate" or "final"; string
-       PY_RELEASE_SERIAL # the 3; an int
+      (PY_MAJOR_VERSION, # the major version number,"2" in "2.1.0a3"; an integer
+       PY_MINOR_VERSION, # the minor version number,"1" in "2.1.0a3"; an integer
+       PY_MICRO_VERSION, # the micro version number,"0" in "2.1.0a3"; an integer
+       PY_RELEASE_LEVEL, # "alpha", "beta", "candidate", or "final"; a string
+       PY_RELEASE_SERIAL, # the release serial number,"3" in "2.1.0a3"; an integer
```

Comment:
I think the original was clearer, we don't need extra words here.

Reply:
Agreed.

BTW, the names of the macros can’t change now, but in other text we try to avoid confusion about the versioning scheme and compatibility expectations by avoiding major/minor and talking about feature release and bugfix release. So best to not change here!  (except maybe two spaces before comment? but no need for comma after the last item)

Also, `integer` is a general term, but `int` a precise python type, so this change loses precision.